src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -341,6 +342,12 @@ def get_beacon_config(
         max_cpu=bn_max_cpu,
         min_memory=bn_min_mem,
         max_memory=bn_max_mem,
+        labels={
+            "ethereum-package-client": "lighthouse",
+            "ethereum-package-client-type": "beacon",
```

Comment:
lets take out beacon, validator and execution as constants as well; or maybe three different label_makers

`execution_label_maker` / `validator_label_maker`